### PR TITLE
QA Bugs Resolved along with the changes for permission set

### PR DESCRIPTION
**Pull request by srikanthv02**
@revinjacob : QA bugs - CAT -1780, 1804,1805 has been resolved.(along with the permission set changes)

CAT  - 1802 i am reopening it again(its on my name) as the code needs to be changed for the condition that will satisfy the save. Shouldn't take much time to resolve and put the alert message.

Import By IP is failing . Please ask someone to test it again.

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js [hunk 1]
```diff
@@ -127,6 +127,7 @@
                             if (selectedList && selectedList.length) {
                                 for (var i = 0; i < selectedList.length; i++) {
                                     taskJSON.assignTasks.push(selectedList[i].data._id);
+                                    $scope.isNewTaskPageLoading = false;
                                 }
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: What is this doing inside a loop?

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js [hunk 2]
```diff
@@ -256,6 +257,7 @@
                             for (var i = 0; i < data.length; i++) {    
                                 if (items.taskConfig.assignTasks[j] === data[i]._id) {
                                     selectorList.push(data[i]);
+                                    $scope.isNewTaskPageLoading = false;
                                 }
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: What is this doing inside a loop?

File: client/htmls/private/index.html [hunk 3]
```diff
@@ -1105,6 +1105,15 @@ <h4 class="modal-title" class="titleCFTDocker" id="myModalLabel"></h4>
...
+} else{
+  //Angular Change - condition added for manual refresh error when the user is in some settings page and refreshes it.
+  var tabSelected = $('#settingsNew').attr('id');
+  if (tabSelected) {
+      var url = window.location.href;
+      if (url.indexOf('#ajax/Settings/') !== -1) {
+          window.location = '#ajax/Settings/Dashboard.html';
+      }
+  }
 }
```
Comment (revinjacob) [inline client/htmls/private/index.html]: Now that there is an else block as well, swap the code blocks and check for positive (isAngularIntegration)

**Comment by srikanthv02** [comment]
@revinjacob : The review comments have been taken care.

**Pull request merged**